packages/trader/src/Assets/Trading/Categories/trade-categories-gif.jsx

```diff
@@ -63,7 +68,12 @@ const TradeCategoriesGIF = ({ category, selected_contract_type }) => {
         case 'turboslong':
             return <ImageTurbos />;
         case 'vanilla':
-            return <ImageVanilla />;
+            return (
+                <ContractTypeDescriptionVideo
+                    selected_contract_type={selected_contract_type}
+                    data_testid='dt_description_video'
```

Comment:
@kate-deriv no need to pass this prop from here (same for line 60) :) please make this prop optional and pass it only from test

Reply:
Done, thank you